ProposedChangesToSpecification.md

```diff
@@ -30,3 +30,145 @@ for storing B1+ fielmaps.
+* Unless the pulse sequence is exclusively associated with a specific qMRI application (e.g. `MP2RAGE`), sequence names are NOT used as `grouping suffixes`.
+* `Grouping suffixes` MUST be used in conjuction with at least one of the `<indexable_metadata>-<index>` and `acq-<label>` key-value pairs. If existing `<indexable_metadata>-<index>` and `acq-<label>` key-value pairs are not enough to describe a new qMRI method, additional request is needed to extend those lists.     
+
+*** 
+
+* `Designation suffixes for qMRI maps` MUST attain a clear description of the parameter that they contain, **including the units**. If available, hyperlinks to example applications and/or more detailed descriptions are highly encouraged.
+
+* If there exist a `designation suffix for qMRI maps` that relates to one or many `grouping suffix`, all the relevances MUST be indicated in the description of the `designation suffix for qMRI map` by the _"Can be generated by: "_ expression.
+
+*** 
+
+* The list of available `suffixes for conventional MRI contrasts` is immutable: `T1w`,`T2w`,`PDw`,`T2starw`. 
+
+***
+
+
+## Indexable Metadata 
+
+### Proposed Change 
+
+### Justification 
+
+## Sidecar JSON files of qMRI maps
+
+Which metadata fields must be included. Some of these fields are to be inherited by the input files and some of them are to be populated by the estimation software. Right now: 
+
+* BasedOn --> List of files gruoped by an grouping suffix + (optional) field maps. 
+* MagneticFieldStrength
+* Manufacturer
+* ManufacturerModelName
+* InstitutionName 
+* PulseSequenceType
+* PulseSequenceDetails
+* EstimationPaper
+* EstimationAlgorithm
+* EstimationSoftwareName
+* EstimationSoftwareVersion 
+* EstimationSoftwareLanguage
+* EstimationSoftwareEnv 
+
+## Sidecar JSON files of grouping suffixes 
+
+Mention about root JSON + repeating JSON file convention.
+Which root JSON metadata fields are optional/required. 
+
+
+## Legacy suffixes for anatomical data
+### Proposed Change 
+Moves some suffixes for `anatomical data` to a new table called "LEGACY suffixes". These suffixes are still valid BIDS format,
+but we reccomend not to use them anymore. They are ambiguous and/or inconsistent.
+
+### Justification 
+The following suffixes are moved to the legacy table:
+ * **`_T2star`**: what is meant is T2\*-_weighted_ data. To be consistent with the suffixes `T1w`, `T2w` and `PDw`, we want
+to use the suffix `_T2starw` for this.
+ * **`_FLAIR`**: This is a sequence name rather than a contrast or a way of grouping quantitative maps. We prefer to use
+`_T2w`, `_PDw`, or `_MESE` depending on the use case.
+ * **`_FLASH`**: This is a sequence name rather than a contrast or a way of grouping quantitative maps. We prefer to use
+`_T2starw`, `_PDw`, or `_GREME` depending on the use case.
```

Comment:
```suggestion
`_T2starw`, `_PDw`, or `_MEGRE` depending on the use case.
```

Reply:
It would be really good to get some highly-clinically oriented people to comment on this.  For white matter lesions (whether vanilla white matter hyperintensities or multiple sclerosis lesions) particular, FLAIR is so ubiquitous that it’s important to spell out exactly what is recommended (and that there is buy-in from the community that uses these scans).

In particular, I have never heard of a PD-weighted FLAIR image.  A concrete replacement for `_FLAIR` as typically encountered should be offered.  I think this is something like:

```
For a typical FLAIR with T2-weighting but suppression of the CSF, 
we recommend `acq-FLAIR_T2w`.
```

Conversely, I’ve never heard of FLASH being used for T2 weighting, no less T2star... AFAIK, in its usual instantiation, it’s used for T1-weighted imaging.  However, here, unlike FLAIR, FLASH just speeds things up and doesn’t change the contrast, so maybe it’s not so important to convey this. (Not 100% confident on this).

**Can I perhaps make a radical suggestion**:  As I understand it, the proposal to remove FLAIR and FLASH are not specific to the remit of BEP001, right?  They are rather part of a general revision/cleaning of the BIDS standard.  *UNLESS* these changes are key to the multi-echo use case, can I propose that you create a *separate* PR, distinct from BEP001, that can get a fuller airing in the BIDS community, on the future of sequenced-named modality labels?  BIDS was not crafted 20 years ago, but just a few years ago, and a wide community of people thought that, despite the slight inconsistency, there was merit in using these particular labels.

@KirstieJane, @Gilles86, @Remi-Gau, @franklin-feingold... what do you think?

Reply:
I am concerned (not opposed) to dropping support of FLAIR and FLASH support in modalities supported. I find these sequences are prevalent (as you mentioned especially in the clinical world). I feel raising a larger community discussion may be best to hear more opinions on this. I do not believe this could uptaken until BIDS 2.0 (due to backward compatibility) anyways, but I think great to hear from more of the community on this.

Reply:
@Gilles86, @agahkarakuzu and I are just chatting about this now - sorry to have dropped the thread for so long.

@nicholst - you've made a strong case for keeping FLAIR in the list of common scan acquisitions. We might tag you in the future for help adding a description to make it clear that we recommend using `FLAIR` only when the scan is T2-weighted with suppression of the CSF.

We'd like to still move FLASH to the legacy table because there are so many different flavours of this acquisition that it is impossible to confidently combine them across datasets. ie if data set 1 has a FLASH scan, there's little chance that it will look the same - have the same contrast - as the FLASH scan in data set 2.

So proposal: keep FLAIR in a prominant position, move FLASH, correct the labels that should have had `w`s after them going forwards (and put the non-`w`-d labels in the legacy table with the same meaning).